src/tokens/blockchain.service.ts

```diff
@@ -94,7 +96,13 @@ export class BlockchainConnectorService {
         this.logger.warn(
           `${request?.path} <-- HTTP ${response?.status} ${response?.statusText}: ${errorMessage}`,
         );
-        throw new InternalServerErrorException(errorMessage);
+        if (response?.status == HttpStatus.CONFLICT) {
```

Comment:
Highlighting (for the record) that we'll generally only reach this block for a "non-retryable" error. The default configuration for which errors are retryable seems to be only those whose error text matches the regex `.*ECONN.*` (although it can be overridden).

So I think unless you override the retry config to be very generic, 409s should generally bubble up to here, and therefore bubble up to FireFly.